causalnex/structure/notears.py

```diff
@@ -101,6 +103,8 @@ def from_numpy(
 
     # n examples, d properties
     _, d = X.shape
+    
+    assert_all_finite(X, allow_nan=False)
```

Comment:
```suggestion
    assert_all_finite(X)
```

`allow_nan=False` is already the default, so no need to set it here again, right? :)